modules/custom-css/csstidy/class.csstidy.php

```diff
@@ -406,10 +406,10 @@ function _unicode(&$string, &$i) {
 		$add = '';
 		$replaced = false;
 
-		while ($i < strlen($string) && (ctype_xdigit($string{$i}) || ctype_space($string{$i})) && strlen($add) < 6) {
-			$add .= $string{$i};
+		while ($i < strlen($string) && (ctype_xdigit($string[$i]) || ctype_space($string[$i])) && strlen($add) < 6) {
```

Comment:
Since we are editing this, should we take the opportunity to add the proper spacing inside the square brackets?